HACKING.md

```diff
@@ -0,0 +1,55 @@
+# Pod Checkpointer Operator Hacking
+
+## Local development
+
+It's possible (and useful) to develop the operator locally targeting a remote cluster.
+
+### Prerequisites
+
+* An OpenShift or Kubernetes cluster with at least a master, infra, and compute
+node. An admin-scoped * `KUBECONFIG` for the cluster. The [operator-sdk](https://github.com/operator-framework/operator-sdk).
+
+### Building
+
+To build the operator during development, use the standard Go toolchain:
+
+```
+$ go build ./...
+```
+
+### Container image
+
+To build the operator docker image:
+
+##### Building using operator-sdk
+
+```
+operator-sdk build quay.io/[some_username]/pod-checkpointer-operator
+docker push quay.io/[some_username]/pod-checkpointer-operator
+```
+
+### Running
+
+To run the operator, first deploy the custom resource definitions:
+
+```
+for f in manifests/*.yaml; do oc create -f $f ; done
```

Comment:
should just be able to apply a directory?

Reply:
thanks. fixed in feb1bdd